backend/app.js

```diff
@@ -0,0 +1,35 @@
+const express = require("express");
+const mongoose = require("mongoose");
+const bodyParser = require("body-parser");
+const passport = require("passport");
+require("dotenv").config();
+
+// const users = require("./routes/user");
+const bounty = require('./routes/bounty');
+
+mongoose.connect(process.env.MONGO_DB_LOCAL, { useNewUrlParser: true }).then(
+    () => {console.log('Database is connected') },
+    err => { console.log('Can not connect to the database' + err)}
+);
+
+const app = express();
+// app.use(passport.initialize());
```

Comment:
are we going to be using passport?

Reply:
No, we all really like passport, but Tate was hitting some walls with it, so he moved to Firebase which he implemented nicely, this will be removed in clean up.